tracker-neuralnet/CMakeLists.txt

```diff
@@ -3,9 +3,8 @@ set(host-spec "${CMAKE_SYSTEM_NAME} ${CMAKE_SYSTEM_PROCESSOR} ${CMAKE_SIZEOF_VOI
 if(host-spec MATCHES "^Linux i[3-6]86 4$")
     return()
 endif()
-if(NOT DEFINED opentrack-use-onnxruntime-avx-dispatch)
-    set(opentrack-use-onnxruntime-avx-dispatch 0)
-endif()
+
+set(opentrack-use-onnxruntime-avx-dispatch False CACHE BOOL "Enable dynamic loading and dispatching to avx-enabled onnxruntime")
```

Comment:
It was fine without the cache variable. It's meant to be an implementation detail for someone using the sdk-paths mechanism. It won't even install properly without cmake `install()` for the dll in sdk paths.

This isn't even a properly named cache variable. They use grouping, i.e. `opentrack_` and tend to form some kind of hierarchy, e.g.

`opentrack_tracker-neuralnet_use-avx-dispatch`

Reply:
Yeah, I thought you might say that. Leave it as is then.